README.md

```diff
@@ -1,90 +1,172 @@
 
-* [Web Assembly](bindings/wasm/)
+1. Clone the repository, e.g. through `git clone https://github.com/iotaledger/identity.rs `
+2. Build the repository with `cargo build `
+3. Run your first example using `cargo run --example getting_started `
+
+If you would like to build the [API Reference](https://identity-docs.iota.org/docs/identity/index.html) yourself from source, you can do so using:
+```rust
+cargo doc --document-private-items --no-deps --open
+```
+
+## Example: Creating an Identity
+
+*Cargo.toml*
+```rust
+[package]
+name = "iota_identity_example"
+version = "1.0.0"
+edition = "2018"
+
+[dependencies]
+identity = { git = "https://github.com/iotaledger/identity.rs", branch = "main"}
+smol = { version = "0.1", features = ["tokio02"] }
+smol-potat = { version = "0.3" }
+```
+*main.*<span></span>*rs*
+```rust
+use identity::crypto::KeyPair;
+use identity::iota::{Client, Document, Network, Result, TangleRef};
+
+#[smol_potat::main] // Using this allows us to have an async main function.
+async fn main() -> Result<()> {
 
-## Planned Milestones
+  // Create a DID Document (an identity).
+  let keypair: KeyPair = KeyPair::new_ed25519()?;
+  let mut document: Document = Document::from_keypair(&keypair)?;
 
-At the current state, the framework is in alpha. As the framework matures we expect to support more and more type of applications. We recommend no use in real-world applications until the consumed libraries are audited, but experimentation and Proof-of-Concept projects are encouraged at the different stages.
+  // Sign the DID Document with the default authentication key.
+  document.sign(keypair.secret())?;
+
+  // Create a client to interact with the IOTA Tangle.
+  let client: Client = Client::new()?;
+
+  // Use the client to publish the DID Document to the IOTA Tangle.
+  document.publish(&client).await?;
+
+  // Print the DID Document IOTA transaction link.
+  let network: Network = document.id().into();
+  let explore: String = format!("{}/transaction/{}", network.explorer_url(), document.message_id());
+  println!("DID Document Transaction > {}", explore);
+
+  Ok(())
+}
+```
+*Example output*
+```rust
+DID Document Transaction > https://explorer.iota.org/mainnet/transaction/YARETIQBJLER9BC9U9MOAXEBWVIHXYRMJAYFQSJDCPQXXWSEKQOFKGFMXYCNXPLTRAYQZTCLJJRXZ9999
+```
+The output link points towards the DID Document transaction, viewable through the IOTA Tangle Explorer, see [here](https://explorer.iota.org/mainnet/transaction/YARETIQBJLER9BC9U9MOAXEBWVIHXYRMJAYFQSJDCPQXXWSEKQOFKGFMXYCNXPLTRAYQZTCLJJRXZ9999). You can see the full DID Document as transaction payload.
+
+## Bindings
+
+[Foreign Function Interface (FFI)](https://en.wikipedia.org/wiki/Foreign_function_interface) Bindings of this [Rust](https://www.rust-lang.org/) library to other programming languages are a work in progress (see Roadmap below). Currently available bindings are:
+
+* [Web Assembly](bindings/wasm/)
```

Comment:
Add explicit JavaScript / TypeScript behind the WASM to clarify the language.

Reply:
fixed in 713dd6d0ef8c7248df17a8bf0642c1c2911a3293